influx/execute.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Do actually executes the query to the specified InfluxDB instance hosted at the url argument.
-func (q Query) Do(url string) (*influx.Response, error) {
+func (q *Query) Do(url string) (*influx.Response, error) {
```

Comment:
The fact that the query was not a pointer was by design. It let us reuse the query builder.

```
a := NewQuery().On("").Limit().GroupBy() ...

web := a.Where("container_type=web")
clock := a.Where("container_type=clock")
```

Reply:
OK I see the use case and the pros